arch/sim/Kconfig

```diff
@@ -509,4 +509,9 @@ config SIM_UART3_NAME
 	default "/dev/ttySIM3"
 	depends on SIM_UART_NUMBER >= 4
 
+config SIM_ONESHOT_HOST_TIMER
+	bool "Use a host timer to deliver oneshot events"
+	default n
+	depends on ONESHOT
```

Comment:
should we change to depend on SIM_WALLTIME? if SIM_WALLTIME equals n, the simulator should run as fast as possible. BTW, is it better to name this option to SIM_WALLTIME_SIGNAL and move after SIM_WALLTIME?

Reply:
Make sense, but changing to choice may better:
```
choice
	prompt "..."
	default SIM_WALLTIME_xxx

config SIM_WALLTIME

config SIM_WALLTIME_xxx

config SIM_WALLTIME_SIGNAL

endchoice
```

Reply:
Yes I think the choice is more suitable. Should we default it to SIM_WALLTIME_SIGNAL ?

Reply:
No, the default value has to be SIM_WALLTIME_xxx to keep the compatiblity.